freeCodeCamp/freeCodeCamp
# feat: A/B testconfetti on block donation modal

Checklist:






Closes #XXXXX


Expected behavior:
This pr displays a confetti to 50% of the users when the block donation modal is popped.

To test this pr:
Seed the db, finish a block, and see the block donation modal without a confetti. (control variant)
Seed the db, change your email to foo@foo.com, complete a block, and see the block donation modal with confetti. (B variant)
